### DISCUSSION TITLE
Decide a package manager for website

### DISCUSSION THREAD

**Discussion by GomezIvann** [dxc-technology/halstack-react#1676] [General] [answered]
We have maintained both `yarn` and `npm` as our package managers on the Halstack website for a while. This can be inconvenient:

![image](https://github.com/dxc-technology/halstack-react/assets/44321109/431c0801-d09c-47ab-b495-fc7012dccdb7)

Since both of them do the same, I think we need to decide and stick with just one manager to avoid future issues with our dependencies.

**Comment by kukoc** [comment, marked as answer]
We will go for npm as package manager. NPM it's widely used since it's installed by default along with Node
